src/test/java/org/apache/ibatis/builder/ParameterExpressionTest.java

```diff
@@ -133,16 +126,20 @@ public void shouldIgnoreLeadingAndTrailingSpaces() {
 
   @Test
   public void invalidOldJdbcTypeFormat() {
-    expectedException.expect(BuilderException.class);
-    expectedException.expectMessage(is("Parsing error in {id:} in position 3"));
-    new ParameterExpression("id:");
+    try {
+      new ParameterExpression("id:");
+    } catch (BuilderException e) {
+      e.getMessage().contains("Parsing error in {id:} in position 3");
```

Comment:
@hazendaz ,
Thanks for the work!
I think there needs to be a call to `fail()` in the try clause, otherwise the test passes even when there is no exception thrown.